### ISSUE TITLE
AdTitle for wrapper ad? 

### ISSUE THREAD

**Issue by wroberts15** [tbuchok/vast-xml#34]
Hey just wondering if you think that having the AdTitle in the wrapper ads would be a good addition? 

Again, it's just something small I noticed- no AdTitle is generated in a wrapper ad even if specified...

**Comment by tbuchok** [comment]
up for a pull request on this? would love to see your contributions to the codebase after all these helpful comments!

**Comment by wroberts15** [comment]
Yeah, sounds good to me. Doubt it'll be much, but I'll do that hopefully by tomorrow.

**Comment by wroberts15** [comment]
Realized that I may need to make some more changes to this module in the upcoming weeks. Did you want to reject this pull request? It works for me locally, if you did want to include this

**Comment by denes16** [comment]
Is this repo still alive?

**Comment by tbuchok** [comment]
hi @denes16 - i haven't put any time into this repo in several years. no longer working in the video world. is there something you're looking for assistance on?

**Comment by denes16** [comment]
> hi @denes16 - i haven't put any time into this repo in several years. no longer working in the video world. is there something you're looking for assistance on?

No thanks, but your repo is very useful, and now im using it in a real project, thanks. Its working good, thanks
